webrender/res/prim_shared.glsl

```diff
@@ -727,11 +727,12 @@ TransformVertexInfo write_transform_vertex(RectWithSize instance_rect,
 
 struct ResourceRect {
     vec4 uv_rect;
+    vec2 user_data;
```

Comment:
I'd like to avoid blob fields unless really necessary. Could we just pass this data in a separate resource cache entry, so that it's strongly typed?